documents/tinyengine/api/frontend-api/settings-panel-api.md

```
@@ -0,0 +1,106 @@
+# 设置面板模块API	
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 12

(MD010, no-hard-tabs)